packages/visualizations/addon/models/table.js

```diff
@@ -51,7 +52,15 @@ export default VisualizationBase.extend(Validations, {
    */
   rebuildConfig(request /*, response */) {
     let dimensions = get(request, 'dimensions') || [],
-        metrics = get(request, 'metrics') || [];
+        metrics = get(request, 'metrics') || [],
+        columns = get(this, 'metadata.columns'),
+        columnIndex = keyBy(columns, ({ field, type }) => {
```

Comment:
Can we add some comments here explaining that the key is the dimensionId or metricId or dateTime?